### PR TITLE
moved logic of fragment

### PR DESCRIPTION
**Pull request by Solod**
edit xml

### CODE REVIEW

File: app/src/main/res/layout/fragment_satart.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        android:gravity="center_horizontal"
+        android:text="@string/app_name"
+        android:textColor="#EFD978"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: To colors.xml

File: app/src/main/res/layout/fragment_satart.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#e6234251"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: To colors.xml

File: app/src/main/res/layout/fragment_satart.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        android:textColor="@android:color/white"
+        android:textSize="@dimen/text_size_display_2"
+        android:textStyle="normal" />
+
+    <Button
+        android:id="@+id/btn_claim"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_marginBottom="5dp"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: To dimensions.xml

Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: dimens.xml

File: app/src/main/res/layout/fragment_satart.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        android:textStyle="bold" />
+
+    <TextView
+        android:id="@+id/sub_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/title"
+        android:gravity="center_horizontal"
+        android:text="@string/sub_title"
+        android:textColor="@android:color/white"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: Create style for this textview

File: app/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -2,4 +2,7 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="text_size_display_1">56sp</dimen>
```
Comment (Gwindor) [inline app/src/main/res/values/dimens.xml]: What 1 mean?

File: app/src/main/res/layout/card_claim_mini.xml [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
         android:id="@+id/cv"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        card_view:cardCornerRadius="20dp"
-        card_view:cardElevation="5dp">
+        style="@style/CardView">
 
-        <LinearLayout
+        <android.support.percent.PercentRelativeLayout
+            xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (Gwindor) [inline app/src/main/res/layout/card_claim_mini.xml]: move it to root LinearLayout

File: app/src/main/res/layout/fragment_satart.xml [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        android:gravity="center_horizontal"
+        android:text="@string/app_name"
+        android:textColor="@color/color_header_title"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: move textColor, textSize and textStyle to styles.

File: app/src/main/res/layout/fragment_satart.xml [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+    <Button
+        android:id="@+id/btn_claim"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_margin="@dimen/my_button_margin"
+        android:background="@drawable/button_primary_selector"
+        android:text="@string/claim"
+        android:textColor="@color/color_text_button"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: to styles

File: app/src/main/res/layout/fragment_satart.xml [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    <TextView
+        android:id="@+id/text_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_above="@id/btn_claim"
+        android:layout_below="@id/subtitle"
+        android:gravity="center_vertical|start"
+        android:padding="@dimen/activity_horizontal_margin"
+        android:text="@string/text_title"
+        android:textColor="@android:color/white"
```
Comment (Gwindor) [inline app/src/main/res/layout/fragment_satart.xml]: to styles

File: app/src/main/java/ua/com/jack_killer/badparking/ui/fragment/StartFragment.java [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+    protected void initView(View view) {
+        mButton = (Button) view.findViewById(R.id.btn_claim);
+    }
+
+    protected void initListener() {
+        mButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (mListener != null) {
```
Comment (Gwindor) [inline app/src/main/java/ua/com/jack_killer/badparking/ui/fragment/StartFragment.java]: extract to class method

Comment (Solod) [inline app/src/main/java/ua/com/jack_killer/badparking/ui/fragment/StartFragment.java]: implemented in branch - Issue#22 Сheck populate fields #22

Comment (Gwindor) [inline app/src/main/java/ua/com/jack_killer/badparking/ui/fragment/StartFragment.java]: I do not see any changes in StartFragment.java in #22

Comment (Solod) [inline app/src/main/java/ua/com/jack_killer/badparking/ui/fragment/StartFragment.java]: I'm sorry not to look. corrected this

**Pull request merged**